unimport/refactor.py

```diff
@@ -41,6 +41,32 @@ def get_location(
     ) -> cst._position.CodeRange:
         return self.get_metadata(cst.metadata.PositionProvider, node)
 
+    def get_rpar(
+        self, rpar: cst.RightParen, location: cst._position.CodeRange
+    ) -> cst.RightParen:
+        return (
+            cst.RightParen(
+                whitespace_before=cst.ParenthesizedWhitespace(
+                    first_line=cst.TrailingWhitespace(
+                        whitespace=cst.SimpleWhitespace(
+                            value="",
+                        ),
+                        comment=None,
+                        newline=cst.Newline(
+                            value=None,
+                        ),
+                    ),
+                    empty_lines=[],
+                    indent=True,
+                    last_line=cst.SimpleWhitespace(
+                        value="",
+                    ),
+                )
+            )
+            if rpar and location.start.line != location.end.line
+            else rpar
```

Comment:
Would you mind unwrapping this if/else to the upper level? For a long expression like this, they just make the whole thing unreadable

Reply:
As far as I can see from the annotations, `rpar` will always be non-None value so it will always be `True`. If that is the case, then please remove that redundant condition too.

Reply:
Single line `cst.ImportFrom` ~> `rpar` == None value.
Multi line `cst.ImportFrom` ~> `rpar` == non-None value.

Reply:
+ I'll refactor the if statement.

Reply:
Wait, if it can be `None` shouldn't the type checker would have catch that (since you have to say `Optional[cst.RightParen]` but you said `cst.RightParen`?

Reply:
It will not catch that.

`Optional[cst.RightParen]` == `Union[cst.RightParen, None]`.

Reply:
> It will not catch that.

A simple example;
```py
import libcst as cst

def foo(bar: cst.RightParen) -> None: ...

statement = cst.parse_statement('from x import y')
import_item = cst.ensure_type(statement.body, cst.ImportFrom)
foo(import_item.rpar)
```
```
 $ mypy t.py
t.py:7: error: Argument 1 to "foo" has incompatible type "Optional[RightParen]"; expected "RightParen"
Found 1 error in 1 file (checked 1 source file)
```

I still don't get why it didn't raised a warning in your initial commit, guess we have some ignores/`Any`s floating around. @hakancelik96, would you mind tracing this?

> `Optional[cst.RightParen]` == `Union[cst.RightParen, None]`.

Don't worry, I know what `Optional` means for!